src/fund-defmodel.jl

```diff
@@ -0,0 +1,252 @@
+    socioeconomic.income    => impacttropicalstorms.income
+    climateregional.regstmp => impacttropicalstorms.regstmp
+
+    population.population   => vslvmorb.population
+    socioeconomic.income    => vslvmorb.income
+
+    vslvmorb.vsl                        => impactdeathmorbidity.vsl
+    vslvmorb.vmorb                      => impactdeathmorbidity.vmorb
+    population.population               => impactdeathmorbidity.population
+    impactvectorbornediseases.dengue    => impactdeathmorbidity.dengue
+    impactvectorbornediseases.schisto   => impactdeathmorbidity.schisto
+    impactvectorbornediseases.malaria   => impactdeathmorbidity.malaria
+    impactcardiovascularrespiratory.cardheat    => impactdeathmorbidity.cardheat
+    impactcardiovascularrespiratory.cardcold    => impactdeathmorbidity.cardcold
+    impactcardiovascularrespiratory.resp        => impactdeathmorbidity.resp
+    impactdiarrhoea.diadead             => impactdeathmorbidity.diadead
+    impacttropicalstorms.hurrdead       => impactdeathmorbidity.hurrdead
+    impactextratropicalstorms.extratropicalstormsdead => impactdeathmorbidity.extratropicalstormsdead
+    impactdiarrhoea.diasick             => impactdeathmorbidity.diasick
+    
+    population.population   => impactwaterresources.population
+    socioeconomic.income    => impactwaterresources.income
+    climateregional.temp    => impactwaterresources.temp
+
+    population.population   => impactsealevelrise.population
+    socioeconomic.income    => impactsealevelrise.income
+    ocean.sea               => impactsealevelrise.sea
+    geography.area          => impactsealevelrise.area
+
+    socioeconomic.income            => impactaggregation.income
+    impactheating.heating           => impactaggregation.heating
+    impactcooling.cooling           => impactaggregation.cooling
+    impactagriculture.agcost        => impactaggregation.agcost
+    impactbiodiversity.species      => impactaggregation.species
+    impactwaterresources.water      => impactaggregation.water
+    impacttropicalstorms.hurrdam    => impactaggregation.hurrdam
+    impactextratropicalstorms.extratropicalstormsdam => impactaggregation.extratropicalstormsdam
+    impactforests.forests           => impactaggregation.forests
+    impactsealevelrise.drycost      => impactaggregation.drycost
+    impactsealevelrise.protcost     => impactaggregation.protcost
+    impactsealevelrise.entercost    => impactaggregation.entercost
+    impactdeathmorbidity.deadcost   => impactaggregation.deadcost
+    impactdeathmorbidity.morbcost   => impactaggregation.morbcost
+    impactsealevelrise.wetcost      => impactaggregation.wetcost
+    impactsealevelrise.leavecost    => impactaggregation.leavecost
+        
+    # ---------------------------------------------
+    # Set leftover parameters
+    # ---------------------------------------------
+
+    set_leftover_params!(FUND, parameters)
+
+end #defmodel
+
+end #module
```

Comment:
Actually, remove this whole file, right? We don't want the ``@defmodel`` stuff to show up.